app/src/main/java/firstapp/weatherapp/MainActivity.kt

```diff
@@ -2,10 +2,14 @@ package firstapp.weatherapp
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import android.widget.ImageView
 
 class MainActivity : AppCompatActivity() {
+
+    lateinit var conditionIcon :ImageView
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+    conditionIcon =findViewById(R.id.condition_icon)
```

Comment:
Since you're setting the icon for the ImageView in the XML you don't need to get access to this view yet.